### ISSUE TITLE
There are repeated word/occurrence pairs in the tWL

### ISSUE THREAD

**Issue by cckozie** [unfoldingWord/gateway-edit#134]
v0.2.0 build cbad9c9

This looks like it is probably a content error, and that the occurrence number is incorrect for the second instance of both Isaac and Jacob is incorrect.
![image.png](https://images.zenhubusercontent.com/56e1a9316caf56c9d38d8df5/855051b8-e94a-4cff-a4d6-a90b6eff05e5)

![image.png](https://images.zenhubusercontent.com/56e1a9316caf56c9d38d8df5/a1f478ef-6d95-4d8d-9232-5cd947ad5b57)

**Comment by PhotoNomad0** [comment]
@cckozie - but if you removed the extra occurrences of brother, how would the user see where/how each occurrence of the word is translated?  Also do you just highlight all occurrences of the word in the scripture?  And then how should we handle if some words have differing Original Language Quotes?

**Comment by cckozie** [comment]
@PhotoNomad0 - I'm pretty sure this is fundamentally a content problem. I'm not sure what, if anything, should be done about it in gatewayEdit, but we should probably have a plan.

**Comment by PhotoNomad0** [comment]
@cckozie What is the reference for this example?

**Comment by cckozie** [comment]
@PhotoNomad0 - The reference is Matthew 1:2

**Comment by lrsallee** [comment]
@cckozie Yes, this is a content issue, but we want to note that we didn't create that issue. Whatever script was used to add in the occurrence numbers was not able to do them correctly. 

The question of how to fix it is more complicated. It does not seem efficient for the Content team to go through tens of thousands of these manually to correct them. It seems like there needs to be a software solution that would find all multiple occurrence in the same verse and increment the occurrence number sequentially. 

As a content guy, I have to ask, "How hard can that be…". :)

**Comment by birchamp** [comment]
@RobH123 does your script add in the occurrence number for TWL?

**Comment by RobH123** [comment]
Yes, I checked my code and it indeed attempts to get the correct occurrence number as you'll see with υἱοῦ in Mat 1:1.

If you look carefully at the spelling of the words that are claimed to have the wrong occurrence numbers (and particularly look at the accent on the last vowel), the same word does NOT occur more than once in Mat 1:2 except for ἐγέννησεν which has the correct occurrence numbers on it.

In other words, I think this issue is a side-track. What was the actual problem that led to it?

**Comment by birchamp** [comment]
@cckozie ^^

**Comment by cckozie** [comment]
My bad. Closing the issue.

**Comment by birchamp** [comment]
@RobH123 Do you know if the occurrences field has been checked in the latest script?

**Comment by RobH123** [comment]
Not sure what you mean @birchamp . 

And as for Perry's apparent comment @cckozie, perhaps he was thinking at a different level or asked the wrong question. TWLs are currently in the UHB and UGNT, i.e., they're connected to Heb and Greek words at file level. So also for the new TSV TWLs.

**Comment by birchamp** [comment]
Yes @RobH123 I meant the conversion, but I wanted to be sure the occurence field is reflecting the content team's expectations. I'll have @BincyJ ask at the next Content meeting.

**Comment by lrsallee** [comment]
@birchamp The baseline is that the TW links feed the TW tool in TC. The way that tool functions is that the user is asked to select the rendering in his text for each TW term. If we remove duplicates from the TWL, it will make a mess of the TW check where the terms occur multiple times in a verse. The user won't know to check for each one, or will not know which word in his translation to connect to the TW term that is presented. The TW tool requires that every instance of the terms be included.

**Comment by lrsallee** [comment]
@birchamp A separate issue might be to display only one instance of TW words in a verse in other tools such as the study app. But that would seem to be a display issue for those tools, and should not affect the content of the TWL.
